server-phoenix/lib/helios/announcement.ex

```diff
@@ -20,11 +20,11 @@ defmodule Helios.Announcement do
     from q in query, where: q.location == ^location
   end
 
-  def happen_today(query, _time_zone) do
+  def happen_today(query) do
     today = DateTime.utc_now()
```

Comment:
I think we set this to midnight if the method is answering the question "does this announcement happen today?"

Reply:
Lets add a unit test